### PR TITLE
deleted/created channel fix

### LINKED ISSUE #65: Deleted Channel
the block `deleted channel`  and `created channel` returns channel.channel which returns undefined
this doesnt need fixed immediately but it does need fixed at some point
the current solution is to use id of deleted/created channel and geting channel from that

### PR DESCRIPTION
**Pull request by RedMan13**

### CODE REVIEW

File: src/blocks/events/channel/channelCreate/channel.js [hunk 1]
```diff
@@ -18,6 +18,6 @@ Blockly.Blocks[blockName] = {
 };
 
 Blockly.JavaScript[blockName] = function() {
-    const code = ["channel.channel", Blockly.JavaScript.ORDER_NONE];
+    const code = ["channel", Blockly.JavaScript.ORDER_NONE];
     return code;
 };
```
Comment (Malix-Labs) [inline src/blocks/events/channel/channelCreate/channel.js]:
```suggestion
};

```

File: src/blocks/events/channel/channelDelete/channel.js [hunk 2]
```diff
@@ -18,6 +18,6 @@ Blockly.Blocks[blockName] = {
 };
 
 Blockly.JavaScript[blockName] = function() {
-    const code = ["channel.channel", Blockly.JavaScript.ORDER_NONE];
+    const code = ["channel", Blockly.JavaScript.ORDER_NONE];
     return code;
 };
```
Comment (Malix-Labs) [inline src/blocks/events/channel/channelDelete/channel.js]:
```suggestion
};

```

**Review by Malix-Labs — CHANGES_REQUESTED** [review]

**Review by Malix-Labs — DISMISSED** [review]

**Comment by Malix-Labs** [comment]
@RedMan13 did you tested it?

**Comment by RedMan13** [comment]
> @RedMan13 did you tested it?

it was already tested
and i checked the docs for d.js, it confirms the fix is correct

**Comment by RedMan13** [comment]
in short no testing needed since its so small of a change a quick look into the docs gives everything needed

**Comment by Malix-Labs** [comment]
- [Comment Reply](https://github.com/scratch-for-discord/Web-Application_Frontend/pull/147#issuecomment-1370668059) from @RedMan13:
  > [[…]](https://github.com/scratch-for-discord/Web-Application_Frontend/pull/147#issuecomment-1370668059) no testing [[…]](https://github.com/scratch-for-discord/Web-Application_Frontend/pull/147#issuecomment-1370668059)

Could you test it anyway please?
Same behavior required before any master merging

**Comment by RedMan13** [comment]
> * [Comment Reply](https://github.com/scratch-for-discord/Web-Application_Frontend/pull/147#issuecomment-1370668059) from @RedMan13:
>   > [[…]](https://github.com/scratch-for-discord/Web-Application_Frontend/pull/147#issuecomment-1370668059) no testing [[…]](https://github.com/scratch-for-discord/Web-Application_Frontend/pull/147#issuecomment-1370668059)
> 
> Could you test it anyway please? Same behavior required before any master merging

wdym "same behavior" the orignal behavior was incorrect

**Comment by Malix-Labs** [comment]
- [Comment Reply](https://github.com/scratch-for-discord/Web-Application_Frontend/pull/147#issuecomment-1370675374) from @RedMan13:
  > wdym "same behavior" [[…]](https://github.com/scratch-for-discord/Web-Application_Frontend/pull/147#issuecomment-1370675374)

I am talking about the required testing before merging into a production-ready branch.

**Comment by RedMan13** [comment]
> * [Comment Reply](https://github.com/scratch-for-discord/Web-Application_Frontend/pull/147#issuecomment-1370675374) from @RedMan13:
>   > wdym "same behavior" [[…]](https://github.com/scratch-for-discord/Web-Application_Frontend/pull/147#issuecomment-1370675374)
> 
> I am talking about the required testing before merging into a production-ready branch.

sbut again this change doesnt need testing
if you mean run the site code yes i did that, 0 iregular function
if you mean the exported code no i didnt bc i didnt need to

**Pull request merged**